.github/workflows/check.yaml

```diff
@@ -1,4 +1,3 @@
-name: "PR health"
```

Comment:
Why this change?

Reply:
Because the title makes debugging just harder: 
![Uploading image.png…]()

Reply:
@lahabana i think your image is missing

Reply:
Does it? That's so noisy to me. the awful prefix in front of everything and then `.yaml`. Can't you always jump to Workflow file on the left of the run summary and even see the name at the top of the list of runs for that workflow?